src/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <script src="https://cdn.tailwindcss.com"></script>
+</head>
+<body class="bg-white">
+  
+    <nav class="py-4 bg-purple-800">
+        <div class="container mx-auto flex justify-between items-center">
+            <h1 class="text-white text-2xl font-bold">Marian Aphiare</h1>
+            <ul class="flex space-x-4 text-white">
+                <li><a href="#home">Home</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#projects">Projects</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </div>
+    </nav>
+
+   
+    <section id="home" class="py-5 text-center">
+        <h2 class="text-4xl font-bold">Marian Aphiare</h2>
+        <p class="text-xl text-gray-500">Frontend Developer</p>
+        <img src="/Resources/asset/images/my photo.jpeg" alt="Marian Aphiare" class="center-image">
+    </section>
+    
+    <style>
+       .center-image {
+        display: block;
+        margin: 0 auto;
+        width: 20%;
+        height: 20%;
+      }
+    </style>
+    
```

Comment:
One line space is enough, read through and leave just one line space in-between elements where there is need to